File: pascal/остальное/sergey/Для школьников 2016/Задача 3/3.pas
Var a,b: Longint;
    f: Text;
Begin
  assign(f,'in.txt');
  reset(f);
  b:=0;
  while not eof(f) do
  begin
    read(f,a); write(a,' ');
    if not eof(f) then b:= b xor a;
  end;
  close(f);

  assign(f,'out.txt');
  rewrite(f);
  write(f,b);
  close(f);
End.

File: pascal/остальное/sergey/Для школьников 2016/Задача 3/3.cpp
#include <iostream>
#include <fstream>
using namespace std;
int main()
 { long a, b=0;
   ifstream F;
   F.open("IN.txt",ios::in);
   if (F) {
     while (!F.eof())
     { F>>a;
       if (!F.eof()) b=b^a; } } else cout<<"Файл не найден!"<<endl;
   F.close();
   ofstream f;
   f.open("OUT.txt");
   f<<b;
   f.close(); return 0; }
